openedx_caliper_tracking/tests/tests.py

```diff
@@ -81,11 +81,46 @@ class CaliperDeliveryTestCase(TestCase):
     )
     def test_event_is_not_called_without_settings(self, delivery_mock):
         """
-        If the settings CALIPER_DELIVER_ENDPOINT and CALIPER_DELIVERY_AUTH_TOKEN are
-        not present then the API call shouldn't fire
+        If the settings ENABLE_CALIPER_EVENTS_DELIVERY, CALIPER_DELIVERY_ENDPOINT
+        and CALIPER_DELIVERY_AUTH_TOKEN are not present then the API call shouldn't fire.
+        """
+        if settings.FEATURES.get('ENABLE_CALIPER_EVENTS_DELIVERY'):
+            settings.FEATURES.pop('ENABLE_CALIPER_EVENTS_DELIVERY')
```

Comment:
why this? you should override settings.

Reply:
usually, we start docstring in tests with `Test` or `Verify` for consistency and readability.

Reply:
> why this? you should override settings.

Actually here I am deleting these settings if exists. Before these changes, our tests were failing as we were trying to delete the settings which weren't present in the settings object.